tests/tests/regression_tests.rs

```diff
@@ -839,6 +839,9 @@ swf_tests_approx! {
     (as3_edittext_tab_stops, "avm2/edittext_tab_stops", 1, epsilon = 5.0),
     (as3_edittext_underline, "avm2/edittext_underline", 1, epsilon = 4.0),
     (as3_math, "avm2/math", 1, max_relative = 30.0 * f64::EPSILON),
+    (as3_matrix, "avm2/matrix", 1, @num_patterns = &[
+        Regex::new(r"\(a=(.+), b=(.+), c=(.+), d=(.+), tx=(.+), ty=(.+)\)").unwrap()
```

Comment:
The `.+` patterns could maybe be made lazy by adding a `?`, if it's supported, and isn't the default. It probably won't change the behavior (hopefully), just a slight optimization.

Reply:
Added lazy capturing